src/framework/ui/view/iconcodes.h

```diff
@@ -174,6 +174,11 @@ class IconCode
         LINE_PEDAL_STAR_ENDING = 0xEF87,
         BEAM_FEATHERING_LEFT = 0xEF88,
         BEAM_FEATHERING_RIGHT = 0xEF89,
+        OTTAVA = 0xF40D,
+        PALM_MUTE = 0xF40E,
+        LET_RING = 0xF40F,
+        VOLTA = 0xF410,
+        VIBRATO = 0xF411,
```

Comment:
It would be great if you could keep this enum in numerical order, so if you could put them just before `DIMINUENDO` (further at the bottom of this file).

Reply:
Oh ok. Will do that